kubejs/assets/ftbquests/lang/pl_pl.json

```diff
@@ -0,0 +1,130 @@
+    "chapter.1.quest.36.subtitle": "Daj mi masowe miejsce na jeden przedmiot",
+    "chapter.1.quest.36.title": "Magazyn Tech Reborn",
+"chapter.1.quest.4.description.1": "Stwórz dowolny element narzędzia Tinkers w budowniczym.",
+"chapter.1.quest.4.subtitle": "Każde narzędzie składa się zwykle z trzech lub więcej komponentów, które mogą być łączone w jedno kompletnie narzędzie, spróbuj.",
+"chapter.1.quest.44.description.1": "skrzynki miedziane oferują więcej miejsca na przechowywanie niż stare i nudne skrzynki drewniane!\n\nIstnieją nawet lepsze kategorie szaf do wytworzenia w przyszłości, ale każda produkcja wymaga kategorii poprzedniej. Z kluczem angielskim Tech Reborn możesz schować + Kliknij prawym przyciskiem myszy na jednostce przechowywania, trzymając klucz angielski, aby zebrać wszystkie przedmioty wewnątrz.",
+"chapter.1.quest.44.subtitle": "Daj mi lepsze ogólne przechowywanie",
+"chapter.1.quest.45.description.1": "Te płytki łączą wszystkie bloki Cooking for Blockheads ze sobą, więc użyj ich jako podłogi w swoich kuchniach.",
+"chapter.1.quest.45.title": "Położenie fundamentów.",
+"chapter.1.quest.46.subtitle": "Ważne bloki z funkcjonalnościami.",
+"chapter.1.quest.46.title": "Wszystko, łącznie z zlewem kuchennym",
+"chapter.1.quest.47.subtitle": "Miejsce na przechowywanie i wodę dla składników.",
+"chapter.1.quest.48.subtitle": "Niektóre jedzenie wymaga dodatkowych składników.",
+"chapter.1.quest.48.title": "Dodatkowe udogodnienia",
+"chapter.1.quest.49.subtitle": "Jeśli jesteś typem budowniczym, możesz być zainteresowany tym. Nie tylko każda z tych stacji wygląda cool, ale także oferuje setki możliwych bloków do budowy, w dowolnym kolorze.",
+"chapter.1.quest.49.task.1.title": "Zrozumiałem.",
+"chapter.1.quest.49.title": "Bloki dekoracyjne!",
+"chapter.1.quest.5.description.1": "Teraz, gdy masz narzędzia do kopania, nic nie stoi na przeszkodzie, aby przejść do wyższego poziomu i stworzyć topielnik z smażonych cegł. Do tego czasu, ta encyklopedia będzie aktualizowana wraz z twoim postępem w Tinkers' Construct, więc trzymaj ją pod ręką. Też dożuciliśmy interesującą książkę o gadżetach do eksploracji!\n\n\nPoza tym, czas zacząć drugą połowę "Minecraft"!\n\n\nO, ważne: Naciskając klawisz (V) (domyślny skrót), możesz przełączać się między trybem ataku a trybem użytkowania dla narzędzi kompatybilnych z walką. Idealne dla sytuacji walki.",
+"chapter.1.quest.5.subtitle": "Połącz części narzędzi w Stacji Tinker, aby stworzyć swój pierwszy narzędzie Tinkers'.",
+"chapter.1.quest.50.subtitle": "Oto przepis na twoje pierwsze piwo.",
+"chapter.1.quest.51.subtitle": "Ślizg",
+"chapter.1.quest.52.subtitle": "zaparz piwo, klikając na ten stół z nimi.",
```

Comment:
Missing capital letter at the beginning of this text.

Also, the syntax can be improved.